src/main/java/org/hmhb/authentication/DefaultJwtAuthenticationService.java

```diff
@@ -70,6 +70,14 @@ public String generateJwtToken(
         claims.put("admin", user.isAdmin());
         claims.put("email", user.getEmail());
         claims.put("userId", user.getId());
+        claims.put("displayName", user.getDisplayName());
```

Comment:
All of this info will now be available to the client in $auth.getPayload() (after you login again -- your old token won't have this info yet).